cpu/nrf5x_common/periph/gpio.c

```diff
@@ -38,6 +38,15 @@
 #define PORT_BIT            (1 << 5)
 #define PIN_MASK            (0x1f)
 
+/* Compatibility wrapper defines for nRF9160 */
+#ifdef NRF_P0_S
+#define NRF_P0 NRF_P0_S
+#endif
+
+#ifdef NRF_GPIOTE0_S
+#define NRF_GPIOTE NRF_GPIOTE0_S
+#endif
```

Comment:
```suggestion
#define GPIOTE_IRQn GPIOTE0_IRQn
#endif
```
saves us an `#ifdef` down the line

Reply:
Done.